docs/src/pages/sections/StarterComponentPage/StarterComponentPage.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styles from './StarterComponentPage.module.css'
+import Link from '@docusaurus/Link'
+
+const data = {
+  name: 'Base Store',
+  owner: 'VTEX',
+  description:
+    'Kickoff your store with this boilerplate. This starter ships with the main FastStore configuration files you need to get up and running blazing fast.',
+  features: [
+    'Landing page',
+    'Product page',
+    'Intelligent search',
+    'Filters for categories',
+    'Optimistic shopping cart',
+    'CSS modules abd Tailwind for straight-forward customization',
```

Comment:
I don't believe we're using CSS modules. Also, there's a small typo.

```suggestion
    'CSS modules and Tailwind for straight-forward customization',
```

Reply:
```suggestion
```